genie/example_filetype_format.py

```diff
@@ -208,6 +208,64 @@ def _validate(self, df: pd.DataFrame, **kwargs) -> tuple:
         logger.info("NO VALIDATION for %s files" % self._fileType)
         return errors, warnings
 
+    def cross_validate_ids_between_two_files(
```

Comment:
I still prefer this to be moved into `transform.py` even though it might be used many times.  Mainly because this is a `@staticmethod`